SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -266,9 +267,20 @@ void FLocalDeploymentManager::TryStartLocalDeployment(FString LaunchConfig, FStr
 	FSpatialGDKServicesModule& GDKServices = FModuleManager::GetModuleChecked<FSpatialGDKServicesModule>("SpatialGDKServices");
 	TWeakPtr<SSpatialOutputLog> SpatialOutputLog = GDKServices.GetSpatialOutputLog();
 
-	RuntimeProcess->OnOutput().BindLambda([&, SpatialOutputLog](const FString& Output) {
+	RuntimeProcess->OnOutput().BindLambda([&, SpatialOutputLog](FString& Output) {
+		// Format and output the log to the editor window `SpatialOutputLog`
 		SpatialOutputLog.Pin()->FormatAndPrintRawLogLine(Output);
 
+		// Save the raw runtime output to disk.
+		if (RuntimeLogFileHandle)
+		{
+			// Always add a newline.
+			Output += LINE_TERMINATOR;
```

Comment:
Does this add an empty line as the first line?

Reply:
Nope, should it?

Reply:
Lol no, read this wrong mb